src/mcts/params.h

```diff
@@ -112,6 +112,9 @@ class SearchParams {
 
   int GetMaxOutOfOrderEvals() const { return kMaxOutOfOrderEvals; }
   float GetNpsLimit() const { return kNpsLimit; }
+  
```

Comment:
It's somehow inconsistent regarding where empty lines are inserted in the list of functions.
Consider removing all the empty lines (or grouping them using some logic other than random).

Reply:
Done.